src/main/drivers/resource.h

```diff
@@ -34,7 +50,7 @@ typedef enum {
     OWNER_RX_SPI,
     OWNER_VTX,
     OWNER_TOTAL_COUNT
-} resourceOwner_t;
+} resourceOwner_e;
```

Comment:
Why this change? It's a `typedef` name, not a `enum` name.

Reply:
Strictly speaking this is true. However the use of the `_e` prefix for enum types gives useful information to the reader of code and the practice is already established in iNav and betaflight.

Reply:
Yes, that's true for enums. Let's keep it like this.